python/paddle/fluid/tests/unittests/mkldnn/test_matmul_mkldnn_op.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest, os
+import numpy as np
+from paddle.fluid.tests.unittests.op_test import OpTest, skip_check_grad_ci
+
+
+@skip_check_grad_ci(reason="DNNL's MatMul doesn't implemend grad kernel.")
+class TestDnnlMatMulOp(OpTest):
+    def generate_data(self):
+        self.x = np.random.random((1, 2, 2)).astype("float32")
```

Comment:
Aren't the dimensions too small for CI? I remember a requirement that tensors should have at least 100 elements.

Reply:
@Sand3r-  https://github.com/PaddlePaddle/Paddle/wiki/OP-Test-Input-Shape-Requirements-(English-Version) The requirements wojtek mentioned could be found here